### PR TITLE
Docs fixes based on customer feedback

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/gridlayout/items.md [hunk 1]
```diff
@@ -174,35 +177,41 @@ The `ColumnSpan` parameter defines the how many columns the item will occupy.
 </TelerikGridLayout>
 
 @code {
-    public int? ColumnSpan { get; set; }
+    public int? ColumnSpan { get; set; } = 1;
```
Comment (ntacheva) [inline components/gridlayout/items.md:180]:
```suggestion
    private int? ColumnSpan { get; set; } = 1;
```

File: components/gridlayout/items.md [hunk 2]
```diff
@@ -221,7 +230,7 @@ The `RowSpan` parameter defines how many rows the item will occupy.
 </TelerikGridLayout>
 
 @code {
-    public int? RowSpan { get; set; }
+    public int? RowSpan { get; set; } = 1;
```
Comment (ntacheva) [inline components/gridlayout/items.md:233]:
```suggestion
    private int? RowSpan { get; set; } = 1;
```

File: components/gridlayout/items.md [hunk 3]
```diff
@@ -135,23 +135,26 @@ The `Column` parameter controls in which column the `GridLayoutItem` will reside
 
 ## ColumnSpan
 
-The `ColumnSpan` parameter defines the how many columns the item will occupy. 
+The `ColumnSpan` parameter defines the how many columns the item will occupy.
+
+When you specify the desired `ColumnSpan`, you should also provide the `Column` that will serve as a start position for the item taking several columns.
```
Comment (dimodi) [inline components/gridlayout/items.md:140]: "provide the `Column`" sounds ambiguous and I had to read it several times and guess what it means.

```suggestion
When you set the desired `ColumnSpan`, you should also set the `Column` parameter to specify the start position for the spanned item.
```

**Review by dimodi — APPROVED** [review]

File: components/gridlayout/items.md [hunk 4]
```diff
@@ -174,35 +177,41 @@ The `ColumnSpan` parameter defines the how many columns the item will occupy.
...
-    public int? ColumnSpan { get; set; }
+    private int? ColumnSpan { get; set; } = 1;
 }
 ````
 
 ## RowSpan
 
 The `RowSpan` parameter defines how many rows the item will occupy.
 
+When you specify the desired `RowSpan`, you should also provide the `Row` that will serve as a start position for the item taking several rows.
```
Comment (ntacheva) [inline components/gridlayout/items.md:188]:
```suggestion
When you set the desired `RowSpan`, you should also set the `Row` parameter to specify the start position for the spanned item.
```

**Pull request merged**